dspace/config/modules/storage.cfg

```diff
@@ -0,0 +1,31 @@
+#---------------------------------------------------------------#
+#-----------------STORAGE CONFIGURATIONS------------------------#
+#---------------------------------------------------------------#
+# Configuration properties used by the bitstore.xml config file #
+#                                                               #
+#---------------------------------------------------------------#
+
+# Use the localStore or the s3Store implementation
+assetstore.storename.0 = localStore
+
+# For using a relative path (xx/xx/xx/xxx...) set to true, default it false
+assetstore.s3.useRelativePath = false
+
+## Assetstore S3 configuration, only used if the above configuration
+## is set to s3Store
+
+# S3 bucket name to store assets in, default would generate a bucket
+# based on the dspace host name
+assetstore.s3.bucketName =
+# Subfolder to organize assets within the bucket, in case this bucket
+# is shared. Optional, default is root level of bucket
+assetstore.s3.subfolder =
+
+# please do not use these in production but rely on the aws credentials
+# discovery mechanism to configure them (ENV VAR, EC2 Iam role, etc.)
```

Comment:
Can we provide more information about the recommended approach to setting these? It seems very odd to have configurations which we don't recommend using. But, if we are doing so, we need to provide them with links on how to set these via environment variables, etc.  Or give examples of what environment variables to user to set these.